module/Company/src/Form/JobLabel.php

```diff
@@ -72,16 +72,6 @@ public function __construct(Translator $translator)
         );
     }
 
-    /**
-     * @inheritDoc
-     *
-     * @return array
-     */
-    public function getInputFilterSpecification(): array
-    {
-        return parent::getInputFilterSpecification();
-    }
-
```

Comment:
While this can be removed, I think it makes it clearer to the developer that the `InputFilter` of this form will contain the necessary elements from the `Localisable` form (the checkboxes).

Reply:
The functionality is exactly the same. I found this very confusing mostly. There's no need to add this. Less code is better code.